app/src/main/java/com/github/swent/echo/compose/components/searchmenu/SearchMenuFilters.kt

```diff
@@ -114,40 +111,54 @@ fun SearchMenuFilters(
                 fullCallback()
             }
         )
+    val followedAssociations = listOf("1", "2", "3", "4", "5")
```

Comment:
will need to be replaced by an argument that propagates from the viewmodel, you should already add it to the signature. Same for selectedAssociation and the callback

Reply:
I will implement the changes to the viewmodel obviously as it is my task, but you should already make the signature consistent

Reply:
I can do it if you want, i thought it was part of the integration with the viewmodel since you just have to propagate those values and call .collectAsState.value in Homescreen

Reply:
The thing is it makes more sense to have the correct signature from the start, but I can still do it in my next task that's fine